### PR TITLE
Updated formatting for CKD state data table.

### PR DESCRIPTION
**Pull request by SantoniS**

### CODE REVIEW

File: docs/source/models/gbd2017/risk_attributable_causes/chronic_kidney_disease/index.rst [hunk 1]
```diff
@@ -39,10 +39,10 @@ on the ages and sexes to which the cause applies.
      -
    * - YLL only
      - False
-     -
+     - Severe CKD only for YLL
    * - YLD only
      - False
-     -
+     - Moderate CKD, Severe CKD include YLD
    * - YLL age group start
```
Comment (alibow) [inline docs/source/models/gbd2017/risk_attributable_causes/chronic_kidney_disease/index.rst]: These notes are a little unclear to me - I think more detail might be helpful here!

**Comment by NathanielBlairStahn** [comment]
Hi @SantoniS , it looks like because of the order things were merged after I rearranged the files, CKD and Diabetes Mellitus ended up in the wrong folder (the old one instead of the new one). At some point these should be moved into

`vivarium_research/docs/source/gbd2017_models/risk_attributable_causes`

and the old folder should be deleted.

File: docs/source/models/gbd2017/risk_attributable_causes/chronic_kidney_disease/index.rst (hunk 1 above)
Comment (SantoniS) [reply docs/source/models/gbd2017/risk_attributable_causes/chronic_kidney_disease/index.rst]: Thanks! I'll add.

**Comment by SantoniS** [comment]
@davidbernotas  Please review when you have a chance, thanks!

**Comment by SantoniS** [comment]
@NathanielBlairStahn I was able to move the CKD folder from the old gbd2017 model folder to the new structure folder without any merge conflicts. Let me know if you think this PR can now safely be merged to master.

File: docs/source/models/gbd2017/risk_attributable_causes/chronic_kidney_disease/index.rst (hunk 1 above)
Comment (SantoniS) [reply docs/source/models/gbd2017/risk_attributable_causes/chronic_kidney_disease/index.rst]: @alibow I ended up removing this information from the Restrictions table and added more explanation to the 'Vivarium Modeling Strategy' section instead. Please let me know if it's a clearer way to frame and has sufficient detail! Thank you.

Comment (SantoniS) [reply docs/source/models/gbd2017/risk_attributable_causes/chronic_kidney_disease/index.rst]: @alibow I'm going to go ahead and merge my branch to master but can create a new PR if we find my update isn't descriptive enough. I'll reach out to you in coworking today!

**Pull request merged**